.github/deploy-prod.yml

```diff
@@ -0,0 +1,44 @@
+name: Docker Build & Push and Deploy to pear-prod
+
+on:
+    push:
+        branches: [release]
+
+jobs:
+    path-context:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+            - name: Set up Docker Buildx
+              uses: docker/setup-buildx-action@v1
+            - name: Login to DockerHub
+              uses: docker/login-action@v1
+              with:
+                  username: ${{ secrets.DOCKER_USERNAME }}
+                  password: ${{ secrets.DOCKER_PASSWORD }}
+            - name: Get SHA
+              id: vars
+              run: echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+            - name: Docker Build & Push
+              uses: docker/build-push-action@v2
+              with:
+                  context: ./
+                  file: ./Dockerfile
+                  push: true
+                  tags: cornellappdev/pear-django:${{ steps.vars.outputs.sha_short }}
+            - name: Remote SSH and Deploy
+              uses: appleboy/ssh-action@master
+              env:
+                  IMAGE_TAG: ${{ steps.vars.outputs.sha_short }}
+              with:
+                  host: ${{ secrets.PROD_SERVER_HOST }}
+                  username: ${{ secrets.SERVER_USERNAME }}
+                  key: ${{ secrets.PROD_SERVER_KEY }}
+                  script: |
+                      export IMAGE_TAG=${{ steps.vars.outputs.sha_short }}
+                      cd docker-compose
+                      docker stack rm the-stack
+                      sleep 20s
```

Comment:
😬

Reply:
yeah we gotta sleep in case the stack takes a long time to remove 😂